compresso/webserver/frontend/src/components/preview/MultiVideoCompare.css

```
@@ -0,0 +1,261 @@
+.comparison-stage {
+  --stage-ink: #080b0d;
+  --stage-panel: #111619;
+  --stage-line: rgba(255, 255, 255, 0.14);
+  --stage-amber: #f3b33d;
+  overflow: hidden;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)